composer.json

```diff
@@ -15,28 +15,28 @@
         "wiki": "http://www.yiiframework.com/wiki/",
         "irc": "irc://irc.freenode.net/yii"
     },
-    "authors": [
-        {
-            "name": "Qiang Xue",
-            "email": "qiang.xue@gmail.com"
-        }
-    ],
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
+        "php": "^7.4",
         "phpspec/php-diff": "^1.1.0",
+        "hiqdev/composer-config-plugin": "^1.0@dev",
```

Comment:
For a package it is a dev-dependency, right?

Reply:
Yes, AFAIK